Documentation/pointsource.rst

```diff
@@ -60,7 +61,7 @@ The point source needs to be turned on in *parameter.par* file.
   FileName=’LOH1_source.dat’
   /
 
-The source input file can be found at . Duration of the source is 4
+The source input file can be found at above link. Duration of the source is 4
```

Comment:
```
can be generated using  `this script <https://github.com/SeisSol/Examples/blob/master/WP2_LOH1/generate_LOH_source_type50.py>`__
```

Reply:
Thanks everyone. Now the link is updated and linked to the master branch Examples.